src/edr_pydantic/reference_system.py

```diff
@@ -0,0 +1,6 @@
+from .my_base_model import EDRBaseModel
+
+
+class CRS(EDRBaseModel):
+    name: str  # MAY EPSG code
+    wkt: str  # SHALL Well Known Text
```

Comment:
Keep the comments like this?

Reply:
Improved. I've a few question still about this object. Trying to resolve those with @rosina-derks

Reply:
Discussed with @lukas-phaf and @PaulVanSchayck that this object is not required in the spec. For this reason, the object will be removed.